client/src/pages/EditProfile/EditProfile.tsx

```diff
@@ -0,0 +1,97 @@
+import { CssBaseline, Grid, Typography, Box, Paper } from '@material-ui/core';
+import { FormikHelpers } from 'formik';
+import CircularProgress from '@material-ui/core/CircularProgress';
+import { useAuth } from '../../context/useAuthContext';
+import { useSocket } from '../../context/useSocketContext';
+import { useHistory } from 'react-router-dom';
+import { useEffect } from 'react';
+import useStyles from './useStyles';
+import profileCreate from '../../helpers/APICalls/profile';
+import ProfileEditForm from './EditProfileForm/EditProfileForm';
+import { useSnackBar } from '../../context/useSnackbarContext';
+
+export default function EditProfile(): JSX.Element {
+  const classes = useStyles();
+
+  const { loggedInUser } = useAuth();
+  const { initSocket } = useSocket();
+
+  const history = useHistory();
+  const { updateSnackBarMessage } = useSnackBar();
+
+  useEffect(() => {
+    initSocket();
+  }, [initSocket]);
+  if (loggedInUser === undefined) return <CircularProgress />;
+  if (!loggedInUser) {
```

Comment:
Remember that `undefined` is `Falsy`. So, are you expecting `loggedInUser` to have any other `Falsy` value?